packages/angular/projects/ui-angular/src/lib/components/authenticator/components/confirm-reset-password/amplify-confirm-reset-password.component.ts

```diff
@@ -15,7 +15,7 @@ export class ConfirmResetPasswordComponent {
   @Input() public headerText = translate('Reset your password');
 
   // translated strings
-  public sendCodeText = translate('Send Code');
+  public submitText = translate('Submit');
```

Comment:
Open to your opinion, do you think this is a breaking change?

Technically yes for customers who were translating the "submit" text, but at the same time this is a technically bug.

Reply:
If yes, we do have a pattern for adding backwards compatible translations:

https://github.com/aws-amplify/amplify-ui/blob/331df831fd7ddeb8c6f28b4cb385d23d1aa524be/packages/angular/projects/ui-angular/src/lib/components/authenticator/components/sign-in/sign-in.component.ts#L24-L28

Reply:
+1 for supporting backwards compatible translations, how do we feel about moving this into the translate function, so that the translate function can also accept a list of strings and if so returns the first one with a match?

Reply:
I agree we should try to make this as backwards compatible as possible. However, since `Send Code` is a string that could appear elsewhere in the Authenticator, we would be defaulting to whatever a user translates for that string instead of `'Submit'`.

In other words, if a user translates `Send Code`, they can never translate the `Submit` text for confirm-reset-passwrd.